customfiles/14_0_INSTALLERCONFIG.sample

```diff
@@ -214,6 +214,9 @@ firstboot_depenguin_run()
 	sleep "${sleep_secs}"
 	for intf in $firstboot_depenguin_interfaces; do
 		echo "$intfs" | grep -Eq "^${intf}$" || continue
+        	if echo "$intf" | grep -Eq "ix"; then
```

Comment:
This also matches ixl interfaces and anything else containing the string "ix". See below for a suggestion